spec/sunomono_test.rb

```diff
@@ -0,0 +1,366 @@
+            to include('android_screen_base.rb', 'features', 'screens', 'step_definitions', 'support')
+        expect(Dir.entries("#{@project_name}/features/ios")).
+            to include('ios_screen_base.rb', 'features', 'screens', 'step_definitions', 'support')
+        expect(Dir.entries("#{@project_name}/features/step_definitions")).
+            to include('base_steps.rb')
+        expect(Dir.entries("#{@project_name}/features/support")).
+            to include('env.rb', 'exceptions.rb')
+
+        # Config files
+        expect(Dir.entries("#{@project_name}/config")).
+            to include('cucumber.yml', 'email', 'load_classes.rb', 'scripts')
+
+        # Email files
+        expect(Dir.entries("#{@project_name}/config/email")).
+            to include('template.html')
+
+        # Scripts files
+        expect(Dir.entries("#{@project_name}/config/scripts")).
+            to include('android', 'break_build_if_failed.sh', 'check_if_tests_failed.sh', 'ios')
+        expect(Dir.entries("#{@project_name}/config/scripts/android")).
+            to include('run_tests_all_devices.sh', 'start_emulators.sh', 'stop_emulators.sh')
+        expect(Dir.entries("#{@project_name}/config/scripts/ios")).
+            to include('.', '..', 'build_app.rb', 'build_app.yml', 'devices.txt', 'run_tests_all_devices.sh')
+      end
+    end
+
+    context 'Try creates new project with a invalid argument' do
+      it 'Project will not be generated' do
+        system "suno new '#{@project_name}' invalid argument", :out => File::NULL
+
+        expect(Dir.entries(".")).
+            not_to include("#{@project_name}")
+      end
+    end
+
+    context 'Should return the latest version of sunonomo' do
+      it 'Returns the latest version' do
+        stdout = Open3.capture3("sunomono version")
+        version = "Sunomono Version " + "#{Sunomono::VERSION}"
+
+        expect(version).
+            to include(stdout.first.gsub(/\n/, ""))
+      end
+    end
+
+    context 'Generate Feature' do
+      it 'Generates all files' do
+        system "sunomono new '#{@project_name}'", :out => File::NULL
+
+        Dir.chdir(@project_name)
+
+        feature_name = 'rspec_sunomono'
+
+        system "sunomono generate feature '#{feature_name}'", :out => File::NULL
+
+        Dir.chdir('../')
+
+        expect(Dir.entries("#{@project_name}/features")).
+            to include("#{feature_name}.feature")
+        expect(Dir.entries("#{@project_name}/features/step_definitions")).
+            to include("#{feature_name}_steps.rb")
+        expect(Dir.entries("#{@project_name}/features/android/screens")).
+            to include("#{feature_name}_screen.rb")
+        expect(Dir.entries("#{@project_name}/features/ios/screens")).
+            to include("#{feature_name}_screen.rb")
+      end
+
+      it 'Generate feature in pt' do
+        system "suno new '#{@project_name}'", :out => File::NULL
+
+        feature_name = 'sunomono'
```

Comment:
You use this variable in a lot of `it` with two different values `sunomono` and `rspec_sunomono`.
Any special reason for this?